src/VolleyManagement.Backend/VolleyManagement.Domain/GamesAggregate/Score.cs

```diff
@@ -18,11 +18,19 @@ public Score()
         /// <param name="home">Score of the home team.</param>
         /// <param name="away">Score of the away team.</param>
         /// <param name="isTechnicalDefeat">Indicating whether the technical defeat has taken place.</param>
-        public Score(byte home, byte away, bool isTechnicalDefeat = false)
+        public Score(byte home, byte away, bool isTechnicalDefeat) : this(home, away)
```

Comment:
It's better to call this ctor rather than calling other

Reply:
ok

Reply:
do you mean that is is better to write ctor with 3 parameters, and use it bu this keyword with last parameter false?